detect-phantom.js

```diff
@@ -17,14 +17,12 @@ if (url === undefined) {
 
 var whitelisted_urls = [];
 if (whitelist_source) {
-  fs.exists(whitelist_source, function(exists) {
-    if (exists) {
-      var whitelist_content = fs.read(whitelist_source);
-      whitelisted_urls = whitelist_content.split("\n");
+    if (fs.exists(whitelist_source)) {
+        var whitelist_content = fs.read(whitelist_source);
+        whitelisted_urls = whitelist_content.split("\n");
     } else {
-      console.error("The whitelist file doesn't exist");
+        console.log("The whitelist file doesn't exist");
```

Comment:
It was to check if you guys were being careful or not when reviewing :)

Or because I was confusing myself on something else, tried many many things and missed this when cleaning up my mess... You pick!